src/components/Card/Card.scss

```
@@ -0,0 +1,74 @@
+.card {
+  max-width: 306px;
+  height: 502px;
+  
+  * + * {
+    padding-left: 47px;
+    padding-right: 47px;
+  }
+
+  & h1 {
+    color: white;
+    font-size: 32px;
+    margin-top: 15px;
+    margin-bottom: 15px;
+  }
+
+  & p {
+    padding-top:15px;
+    font-size: 18px;    
+    line-height: 1.3em;
+  }
+
+  & button {
+    margin-top: 120px;
+    margin-left: 47px;
+  }
+
+  &--sedan {
+    background-color: #E28525;
+    
+    & p {
+      color: #F8C885;
+    }  
+  }
+
+  &--suvs {
+    background-color: #016972;
+
+    & p {
+      color: #A0E2EE;
+    }
+  }
+
+  &--luxury {
+    background-color: #00403F;
+
+    & p {

```

same here https://github.com/MarcelloAragoni/mentor/pull/2/files#r844145043

```suggestion
    p {
```